cloudify_awssdk/ec2/resources/eni.py

```diff
@@ -145,6 +147,15 @@ def create(ctx, iface, resource_config, **_):
             subnet_id or \
             targ.target.instance.runtime_properties.get(EXTERNAL_RESOURCE_ID)
 
+    groups = params.get(SEC_GROUPS, [])
+    for targ in utils.find_rels_by_node_type(ctx.instance, SEC_GROUP_TYPE):
+        group_id = \
+            targ.target.instance.runtime_properties.get(
+                EXTERNAL_RESOURCE_ID)
+        if group_id not in groups:
```

Comment:
If the ``group_id`` is ``None`` should we worry about it ? like ignore any NONE value